client/src/components/new-request/analyze-pending.vue

```diff
@@ -8,11 +8,12 @@
     </template>
     <template v-slot:content>
       <v-row no-gutters justify="space-between" align-content="space-around">
-        <NameInput class="mt-3" />
+        <NameInput class="mt-3" :is-search-again="true"/>
```

Comment:
I'm not certain, but I think if the value isn't reactive then you don't need the colon. LMK!

Reply:
I'm glad you asked, i did try this before. Ie `is-search-again` as a property and it didn't register. I didn't try `is-search-again="true"` with no colon however.

Reply:
Yeah, I know Vuetify supports the prop name alone. I never looked up how they did it.

Reply:
`<NameInput class="mt-3" is-search-again="true"/>` it appears to work without the colon. Removed 👍

Reply:
Noice!